src/Redux/Books/Books.js

```diff
@@ -0,0 +1,43 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [
+    {
+      id: 1,
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+    },
+    {
+      id: 2,
+      title: 'Dune',
+      author: 'Frank Herbert',
+    },
+    {
+      id: 3,
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+    },
+  ],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBooks(state, action) {
+      if (action.payload) {
+        state.books.unshift(action.payload);
+      }
+    },
+    deleteBooks(state, action) {
+      if (action.payload) {
+        const temp = state.books.filter((book) => book.id !== action.payload);
+        /* eslint-disable */
```

Comment:
- **Please solve this Linters manually, Don't disable the Linter.**
- **Creates a reducer and exports it as default:** At the end of the day, the new book must be added to an array also can be removed from an array